dev-install-locally-no-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+./gradlew clean install -x test
```

Comment:
No :-) This was the most convenient thing for me :-) What shell scripts? Can you point out where I could add it?